frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -87,7 +80,12 @@ export const HeaderContainer = styled.header`
 
 export const ParametersAndCardsContainer = styled.div`
   flex: auto;
+  overflow-y: ${({ shouldMakeDashboardHeaderStickyAfterScrolling }) =>
```

Comment:
nit: shorter parameter name? :)

Reply:
The other approach in this file is when we specify what we pass instead. e.g. `isEditing or `isEditingOrSharing` which won't work with what we're passing here. `!isFullscreen && (isEditing || isSharing)` so I think it's better to document what we want this flag to mean.

I'm open to new names. But this is the most concise version I could come up with at the time. I think the weirdness with the name would be getting away once we start making dashboard UI header + sticky components more consistent. You can see [in my other comment](https://github.com/metabase/metabase/pull/31210#discussion_r1221121046) that we have quite different variations for dashboard headers. Notably different elements will be sticky depending on whether we're opening a subscription sidebar or the info sidebar.